internal/servicedeployer/_static/terraform_deployer.yml

```diff
@@ -13,3 +13,4 @@ services:
     volumes:
       - ${TF_DIR}:/stage
       - ${TF_OUTPUT_DIR}:/output
+      - ${SERVICE_LOGS_DIR}:/tmp/service_logs/
```

Comment:
Required to create files that could be accessible by the Elastic Agents.

This folder/volume is also used by the services (e.g. nginx):
https://github.com/elastic/elastic-package/blob/48ad94bd16cb08416b9ba6d30b71b716656faecc/test/packages/parallel/nginx/_dev/deploy/docker/docker-compose.yml#L8